src/main/java/mops/termine2/controller/TermineNeuController.java

```diff
@@ -224,49 +234,50 @@ public String uploadTermineCSV(@RequestParam("file") MultipartFile file, Princip
 					new InputStreamReader(file.getInputStream(), StandardCharsets.UTF_8))) {
 					
 					List<String[]> termineEingelesen = csvReader.readAll();
+					if ("DATUM".equals(termineEingelesen.get(0)[0])) {
+						termineEingelesen.remove(0);
+					}
 					
 					TerminFormatierung terminFormatierung =
 						new TerminFormatierung(termineEingelesen);
-					terminFormatierung.pruefeFuerJedenTerminGueltigesFormat(
-						termineEingelesen, terminFormatierung.getDateTimeFormatter());
-					terminFormatierung.pruefeObExistent(termineEingelesen);
 					
-					if (!terminFormatierung.pruefeObInZukunft(termineEingelesen,
+					if (!terminFormatierung.pruefeObGueltigesFormat(
+						termineEingelesen, terminFormatierung.getDateTimeFormatter())) {
+						m.addAttribute("message",
+							"Alle Termine müssen im Format "
+								+ "'TT.MM.JJJJ,HH:MM' übergeben werden und "
+								+ "sollten existente Daten sein.");
+						m.addAttribute("error", true);
+					} else if (!terminFormatierung.pruefeObInZukunft(termineEingelesen,
```

Comment:
Könnte man das Überprüfen des Formats nicht gut in eine eigene Klasse auslagern ?